tests/PySys/monitoring/monitoring_with_simulated_messages/pysystest.xml

```diff
@@ -8,7 +8,8 @@
 		<purpose>
 			<![CDATA[ ]]>
 		</purpose>
-	</description>
+		 </description>
+		<skipped reason="Fails due to retained message. See #1081"/>
```

Comment:
Is this test really [affected by retained messages](https://github.com/thin-edge/thin-edge.io/issues/1081) ?
The feature under test is unrelated to retained messages.

Reply:
Hi, sorry for the delayed answer. For me, the test fails due to an additional line in the output from the retained message. The text fix should be easy by adjusting the topic. I can fix it, no Problem. @gligorisaev What do you think?